src/abstract-api-adapter.coffee

```diff
@@ -0,0 +1,67 @@
+request = require 'request'
+try
+	{ Adapter } = require 'hubot'
+catch
+	prequire = require 'parent-require'
+	{ Adapter } = prequire 'hubot'
+
+class AbstractAPIAdapter extends Adapter
+	constructor: ->
+		super
+		@interval = 1000
+		@lastReport = null
+		@clearToPoll = true
```

Comment:
What does `clearToPoll` control? The name isn't obvious to me and I'm wondering if there's a better alternative